frontend/src/packages/drivers/libs/enums/enums.ts

```diff
@@ -0,0 +1,7 @@
+const DriverCreationMessage = {
+  ERROR: 'Unexpected error, driver was not created',
+  SUCCESS: 'Driver was successfully created',
+};
```

Comment:
the error message comes from be, isn't it?

Reply:
Yes it is